src/inmanta/server/config.py

```diff
@@ -63,6 +79,29 @@
 #############################
 # server
 #############################
+server_bind_address = Option(
+    "server", "bind-address", "127.0.0.1", "A list of addresses on which the server will listen for connections", is_list
+)
+server_bind_port = Option("server", "bind-port", 8888, "The port on which the server will listen for connections", is_int)
+
+
+def get_bind_port() -> int:
```

Comment:
why not use Option.predecessor_option
or alternatively, use the option_as_default wrapper 
or extend the option_as_default wrapper  to do deprecation? 

This is an exception to the way options are fetched